src/equations/ideal_glm_mhd_1d.jl

```diff
@@ -277,6 +277,22 @@ end
     λ_max = max(abs(v_ll), abs(v_rr)) + max(cf_ll, cf_rr)
 end
 
+# Calculate estimates for minimum and maximum wave speeds for HLL-type fluxes
+@inline function min_max_speed_naive(u_ll, u_rr, orientation::Integer,
```

Comment:
Is this function still needed for the time step estimation or so? From the comment in the NEWS item I gathered that one must use the `flux_hlle` for ideal MHD now. Should this comment be updated?

Reply:
This is the default estimate which is used when you choose `flux_hll`.

Reply:
But then I am confused. You changed all the elixirs to use `flux_hlle` as I thought this was a requirement now. Is it still possible to use `flux_hll` with ideal MHD and it is just not the "default"?

Reply:
Yes - we previously used the HLLE wave speed estimates for MHD. Thus, he changed all examples to use `flux_hlle` to keep the same setups. `flux_hll` now uses the Davis estimate - as for all other equations.